packages/components/Button/src/Button.types.ts

```diff
@@ -29,24 +29,27 @@ export interface IButtonInfo extends IPressableState {
 }
 
 /**
- * Because state updates are coming from the touchable and will cause a child render the button doesn't use
- * changes in state value to trigger re-render.  The values inside inner are effectively mutable and are used
- * for per-component storage
+ * Because state updates are coming from the touchable and will cause a child
+ * render the button doesn't use changes in state value to trigger re-render.
+ * The values inside inner are effectively mutable and are used for
+ * per-component storage
  */
 export interface IButtonState {
   info: IButtonInfo;
 }
 
-export interface IButtonTokens extends FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {
+export interface IButtonTokens extends FontTokens, IForegroundColorTokens,
+                                       IBackgroundColorTokens, IBorderTokens {
   /**
    * The amount of padding between the border and the contents.
    */
-  contentPadding?: number | string;
+  contentPadding?: number|string;
```

Comment:
Where is this prettier setting coming from? Actually in general it looks like your prettier settings might be out of date in some way, are you using the ones from the repo?

Reply:
> Where is this prettier setting coming from? Actually in general it looks like your prettier settings might be out of date in some way, are you using the ones from the repo?

There are the settings from the repo.   Every time I hit Save in Visual Studio Code it rewrites the file based on the settings in `.vscode/settings.json`.   I'm surprised I'm the only one impacted by this.   In a previous PR I tried turning off the `"editor.formatOnSave": true` setting but was asked to revert that change: https://github.com/microsoft/fluentui-react-native/pull/418#discussion_r480435786